src/ClusterBootstrap/cloud_init_deploy.py

```diff
@@ -267,6 +267,25 @@ def load_default_config(config):
     return config
 
 
+def create_basic_auth_4_k8s(overwrite=False):
+    basic_auth = load_basic_auth()
+    if basic_auth is None or overwrite:
+        os.system("rm -f ./deploy/k8s_basic_auth.yml || true")
```

Comment:
why `|| true`?

Reply:
because utils.gen_SSH_key did it that way, I don't know whether there's a scenario that I am not aware of, so just followed that